docs/user-guide/global-configurations/gitops.md

```diff
@@ -1,47 +1,51 @@
 # Gitops
 
 ## Why Devtron takes Gitops Configuration?
-Devtron uses GitOps and stores configurations in git; Git Credentials can be entered at `Global Configuration > GitOps` is used by Devtron for configuration management and storing desired state of the application configuration. 
+Devtron uses GitOps and stores configurations in git; Git Credentials can be entered at `Global Configuration > GitOps` which is used by Devtron for configuration management and storing desired state of the application configuration. 
+In case GitOps is not configured, Devtron cannot deploy any application or charts. 
 
 
-In case GitOps is not configured, Devtron cannot deploy any application or charts. Areas impacted by GitOps are:
+Areas impacted by GitOps are:
 
 * Deployment Template, [click here](https://docs.devtron.ai/user-guide/creating-application/deployment-template) to learn more.
 * Charts, [click here](https://docs.devtron.ai/user-guide/deploy-chart) to learn more.
 
 
 ## Add Git Configuration
 
-Select the Gitops section of global configuration. At the top of the section, two Git providers are available.
+Select the Gitops section of global configuration. At the top of the section, three Git providers are available.
 
 * **GitHub**
 * **GitLab**
+* **Azure**
 
-![](../../.gitbook/assets/gc-gitops-git.png)
+![](../../.gitbook/assets/gc-gitops-provider.png)
 
 Select one of the Git providers. To add git account, You need to provide three inputs as given below:
-1. Git Host
-2. Github Organization Name / Gitlab Group id 
-3. Git access credential
+1. Git Host / Azure DevOps Organisation Url <br />
+2. Github Organization Name / Gitlab Group id / Azure DevOps Project Name <br />
+3. Git access credential <br />
 
 ### 1. Git Host: 
 
-This field is filled by default, Showing the URL of the selected git providers. For example- https://github.com for Github, https://gitlab.com for GitLab.
+This field is filled by default, Showing the URL of the selected git providers. For example- https://github.com for Github, https://gitlab.com for GitLab, https://dev.azure.com/ for Azure. 
```

Comment:
add
Please change them if they are not the url you use.

Reply:
<img width="269" alt="Screenshot 2021-05-24 at 3 24 22 PM" src="https://user-images.githubusercontent.com/61836271/119330701-24068e80-bca4-11eb-9be0-dd2e50369974.png">
we are using this link 
@pghildiy

Reply:
yes we are using this link `host: "https://dev.azure.com/"`
@pghildiy

Reply:
added